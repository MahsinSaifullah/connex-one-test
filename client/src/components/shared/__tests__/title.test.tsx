import { render } from '@testing-library/react';

import { Title } from '../title.component';

describe('Title Component', () => {
  it('should render title component correctly', () => {
    const { asFragment } = render(<Title title="test title" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
