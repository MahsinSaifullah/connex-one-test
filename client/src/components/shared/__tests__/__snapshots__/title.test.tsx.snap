// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component should render title component correctly 1`] = `
<DocumentFragment>
  <h2
    class="text-lg sm:text-2xl font-bold tracking-widest undefined"
  >
    test title
  </h2>
</DocumentFragment>
`;
